### PR TITLE
initial push for portfolio Details

### PR DESCRIPTION
**Pull request by DGabriel-CCPO**

### CODE REVIEW

File: src/components/ATATTextField.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
         class="form-field-label my-1"
         :for="id + '_text_field'"
       >
-        Text Field label
+        {{ id }}
```
Comment (tonysumner-ccpo) [inline src/components/ATATTextField.vue:9]: If an id is used that contains multiple words like "First Name", additional logic will be needed to replace the space with an underscore, resulting in "first_name".  Send over the prop called fieldLabel, and use that.

File: src/main.ts [hunk 2]
```diff
@@ -14,14 +14,16 @@ import ATATHeaderNav from "./components/ATATHeaderNav.vue";
...
 import SecurityBanner from "./components/SecurityBanner.vue";
 import StyledFields from "./components/StyledFields.vue";
 import USAGovBanner from "./components/USAGovBanner.vue";
 
 // wizard
 import ViewPortfolio from "./wizard/Step_0/components/ViewPortfolio/ViewPortfolio.vue";
 import CreatePortfolio from "./wizard/Step_0/components/CreatePortfolio/CreatePortfolio.vue";
-
+Vue.component('atat-text-field', ATATTextField);
+Vue.component('atat-text-area', ATATTextArea);
```
Comment (tonysumner-ccpo) [inline src/main.ts:26]: keep these in alphabetical order.

File: src/wizard/Step1/components/CreatePorfolioForm.vue [hunk 3]
```diff
@@ -2,7 +2,50 @@
...
+    <v-row>
+      <v-col cols="6">
+        <v-form ref="form">
+          <atat-text-field id="Portfolio Name" />
+          <p class="mb-11">
+            Naming can be difficult. Choose a name that is descriptive enough
+            for users to identify the portfolio. you may consider naming based
+            on your organization.
+          </p>
+          <atat-text-area id="Portfolio Description" />
```
Comment (tonysumner-ccpo) [inline src/wizard/Step1/components/CreatePorfolioForm.vue:17]: The word 'Optional' is missing.

File: src/components/ATATTextField.vue [hunk 4]
```diff
@@ -39,10 +43,13 @@ export default class ATATTextField extends VTextField {
   @Prop({ default: false }) private success!: boolean;
   @Prop({ default: false }) private error!: boolean;
   @Prop({ default: "id_is_missing" }) private id!: string;
+  @Prop({ default: "input" }) private label!: string;
```
[the remark this answers is not in the document]
Comment (tonysumner-ccpo) [reply src/components/ATATTextField.vue]: yes, good idea.  changed it to 'Form Field Label'

**Review by jGonzalez-CCPO — APPROVED** [review]

**Pull request merged**